modules/images-configuration-file-hcp.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-configuration-file-hcp_{context}"]
+= Configuring image registry settings for HCP
+
+You can configure image registry settings by editing the `image.config.openshift.io/cluster` custom resource (CR).
+When changes to the registry are applied to the `image.config.openshift.io/cluster` CR, the Machine Config Operator (MCO) performs the following sequential actions:
```

Comment:
I don't think we need this section, but @andreadecorte to confirm.

Reply:
yes. we need to replace with a section that mentions the rollout across all the node pools.
Something like:
any change will rollout the nodes across all the node pools with the new desired config.

Similar to the CLI message

Reply:
@andreadecorte Can you confirm please which lines (is it lines 9 & 10) need to be removed? thanks

Reply:
@andreadecorte which CLI message ? thanks

Reply:
I think this can be resolved.